### DISCUSSION TITLE
Clublog Realtime Upload

### DISCUSSION THREAD

**Discussion by GregTroar** [magicbug/Cloudlog#2642] [General]
Hi,

I am not sure I understand in the wiki how to get the realtime upload of Clublog QSOs.
Which callsign am I supposed to use at the end ?

https://url-to-cloudlog/index.php/clublog/realtime/<what should I put here>

I did try with my Clublog login call but it does not work then tried my other two callsigns and it does not work
I always have: Username unknown.

To be noted that https://url-to-cloudlog/index.php/clublog/upload works just fine.

Thanks

**Comment by GregTroar** [comment]
No one ?
https://<url-and-path-to-cloudlog>/index.php/clublog/realtime/<The-Cloudlog-Username-You-Just-Added-Logins-To>
if the clublog settings I added are f4bpo then I changed <The-Cloudlog-Username-You-Just-Added-Logins-To> to f4bpo but no luck

**Comment by GregTroar** [comment]
Cannot get the clublog live upload to work, is anyone using this function to upload each qso to clublog immediately after it is logged ?

**Comment by phl0** [comment]
You need to use the username of the Cloudlog account you want to use here (and has clublog credentials configured). I guess most people use their callsign to login to Cloudlog. So best is to try that one :)

**Comment by GregTroar** [comment]
ok finally made it work.

Thanks a lot for help
